bitsofgood/pencils-for-success
# feat(ui): connect admin modal to backend endpoints

### Description
- Viewing the admin modal now pulls from the backend endpoint
- Editing the admin username and password now updates the backend
- Backend updates are now reflected in the frontend admin modal

**Related Issues/PRs:** 
- Closes #148

### Test Plan
- Login as an admin
- Click View Admin Info
- Click Edit
- Enter a new valid username and password
- Click update
- Click View Admin Info and confirm that the username has updated
- Logout
- Log back in with new username and password
- Click View Admin Info and confirm the new username
